<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>670</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="year_widget">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>37</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>5</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>사용자 :</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>17</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_13">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>250</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="year_hun">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>34</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>년도 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="hun_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>72</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>헌금명칭 :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="hun_name_widget">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>72</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="fix_hun_detail">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>75</y>
     <width>79</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_22">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>590</y>
     <width>631</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="user_widget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>7</y>
     <width>56</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="huncancel_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>610</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="hun_row_modi_Button">
   <property name="geometry">
    <rect>
     <x>44</x>
     <y>610</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>1줄 수정 저장</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="hap_total">
   <property name="geometry">
    <rect>
     <x>219</x>
     <y>112</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>등록되는 헌금총액 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hap_total_widget">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>112</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="month_hun">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>37</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>월 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="month_widget">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>36</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="week_hun">
   <property name="geometry">
    <rect>
     <x>198</x>
     <y>40</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>몇번째 주 :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="week_widget">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>36</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="hun_detail_combo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>71</y>
     <width>130</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentText">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="modi_hun_tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>621</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <column>
    <property name="text">
     <string>날짜</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>개별코드</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>성명</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>금액</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>통장예입</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="re_calculate_button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>합계금액 재계산</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="remove_row_Button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>610</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>선택행삭제</string>
   </property>
  </widget>
  <widget class="QLabel" name="hun_name_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>35</y>
     <width>65</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>회계 구분</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gubun_name_combo_widget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>36</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="serch_Button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hun_account_modi_Button">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>610</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>선택행 계정 수정 저장</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>month_widget</tabstop>
  <tabstop>week_widget</tabstop>
  <tabstop>gubun_name_combo_widget</tabstop>
  <tabstop>hun_name_widget</tabstop>
  <tabstop>hun_detail_combo</tabstop>
  <tabstop>serch_Button</tabstop>
  <tabstop>modi_hun_tableWidget</tabstop>
  <tabstop>hun_row_modi_Button</tabstop>
  <tabstop>hun_account_modi_Button</tabstop>
  <tabstop>re_calculate_button</tabstop>
  <tabstop>remove_row_Button</tabstop>
  <tabstop>huncancel_button</tabstop>
  <tabstop>year_widget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>huncancel_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>file_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>657</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>658</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serch_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>hun_serch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>522</x>
     <y>65</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>re_calculate_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>re_calculate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remove_row_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>remove_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>604</x>
     <y>658</y>
    </hint>
    <hint type="destinationlabel">
     <x>577</x>
     <y>657</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hun_account_modi_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>hun_account_modi_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>629</y>
    </hint>
    <hint type="destinationlabel">
     <x>168</x>
     <y>633</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hun_row_modi_Button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>row_modi_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>642</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>655</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>row_modi_save()</slot>
  <slot>re_calculate()</slot>
  <slot>file_close()</slot>
  <slot>remove_row()</slot>
  <slot>hun_serch()</slot>
  <slot>hun_account_modi_save()</slot>
 </slots>
</ui>
